src/pybind/mgr/dashboard/controllers/summary.py

```diff
@@ -41,15 +41,43 @@ def _rbd_mirroring(self):
                 warnings += 1
         return {'warnings': warnings, 'errors': errors}
 
+    def _task_permissions(self, name):
+        result = True
+        if name == 'pool/create':
+            result = self._has_permissions(Permission.CREATE, Scope.POOL)
+        elif name == 'pool/edit':
+            result = self._has_permissions(Permission.UPDATE, Scope.POOL)
+        elif name == 'pool/delete':
+            result = self._has_permissions(Permission.DELETE, Scope.POOL)
+        elif name in [
```

Comment:
The required permissions for each type of task should match what is in the original declaration of the controller that creates the task. The `rbd/*` tasks are created/declared in the `controllers/rbd.py` file. 

For instance, the `rbd/flatten` requires update permission and not create.

Reply:
How can we make sure, this is not getting outdated in the future?

Reply:
I think the only way would be to encode the required permissions inside the task object upon task creation.

But that requires a much more complex solution.

Reply:
Changes made: task types now match the required permissions.